uploadNeuroDB/tarchiveLoader

```diff
@@ -604,7 +604,7 @@ if ($valid_study) {
 
     my $sth = $dbh->prepare($query);
     $sth->execute($tarchiveInfo{TarchiveID});
-    my $oldCount = $sth->fetchrow_array;
+    my $oldCount = $sth->rows == 0 ? 0 : $sth->fetchrow_array;
```

Comment:
for me `$sth->rows` = 1 (it still returns one row whether the value is NULL or a number).

Should probably be something like that instead:
`my $oldCount = $sth->fetchrow_array ? $sth->fetchrow_array : 0`